manifests/n/NeumetrixLimited/HttpWatchBasic14/0Setup/14.0.16/NeumetrixLimited.HttpWatchBasic14.0Setup.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: NeumetrixLimited.HttpWatchBasic14.0Setup
+PackageVersion: 14.0.16
+PackageLocale: en-US
+Publisher: Neumetrix Limited
+PackageName: HttpWatch Basic 14.0 Setup

```

```suggestion
PackageName: HttpWatch Basic
```
Should match control panel as closely as possible